Alex garcia lesson1 homework2

Saex05 opened the pull request:
Add car project in the path ...\alex-garcia\lesson-1\homework-2-car-project

Saex05 committed:
Edit file names

Saex05 committed:
add car project in lesson-1/homework-2

J4rv1sX commented on alex-garcia/lesson-1/homework-1-contact-manager/ContactsManager.java:
```diff
@@ -1,32 +1,32 @@
+    }
+
+    // Methods:
+    void addContact(Contact Contact){
```
Please start parameters with lower case

Saex05 replied:
Update: Parameters started with lower case

J4rv1sX commented on alex-garcia/lesson-1/homework-2-car-project/CarProject/src/CarBehavior.java:
```diff
@@ -0,0 +1,126 @@
+    }
+
+    // Methods:
+    void addCar(Car Car){
```
Same here, start with lowercase for Fields, Methods and Parameters

Saex05 replied:
Update: Parameters started with lower case

J4rv1sX commented on alex-garcia/lesson-1/homework-2-car-project/CarProject/src/CarBehavior.java:
```diff
@@ -0,0 +1,126 @@
+
+class CarBehavior {
+    static class Car {
+        String name;
```
I will recommend to just to have a single class called Car, in either case please define your Fields with private access modifier and create access methods is necessary (getters and setters).

Saex05 replied:
Update: Fields defined  with private access modifier and access methods (getters) created in the car project.

Saex05 committed:
1.- Parameters started with lower case
2.- Fields defined  with private access modifier and access methods (getters) created in the car project.
